### ISSUE TITLE
Calling explain_instance twice results in different explanations

### ISSUE THREAD

**Issue by Sebber123** [marcotcr/lime#309]
Hi,

I am calling the following code 

```
explainer = lime.lime_tabular.LimeTabularExplainer(train, feature_names=feature_names, class_names=class_names, categorical_features=cat_indices,categorical_names=categorical_names, 
kernel_width=kernel_width, random_state=0)

exp = explainer.explain_instance(test[i], predict_fn, num_features=5)

exp = explainer.explain_instance(test[i], predict_fn, num_features=5)

```

I understand that minor deviations in weights could occur. However, the explanations produced above result in different features too. I tried to call random seed before I called explain instance, but nothing changes.

```
def explain(instance, predict_fn, **kwargs):
   
    np.random.seed(16)
   
    return explainer.explain_instance(instance, predict_fn, **kwargs)

```

Does anyone know what causes this? This happens for low kernel widths but also occasionally for higher kernel widths.

Thanks in advance!

**Comment by marcotcr** [comment]
Different features could be returned when there are feature that are very close, such that whether one or the other gets picked depends on the random sample.

If you want to set the random seed, you would need to do the following:
```
def explain(instance, predict_fn, **kwargs):
    explainer.random_state = sklearn.utils.check_random_state(16)
    return explainer.explain_instance(instance, predict_fn, **kwargs)
```
